editor/components/post-permalink/style.scss

```diff
@@ -29,6 +29,7 @@
 
 .editor-post-permalink__label {
 	margin: 0 10px 0 5px;
+	font-weight: 600;
```

Comment:
Minor: Could / should we add `<strong>` or `<b>` as a nested element of the label instead?

Reply:
Not sure. `<strong>` adds a semantic meaning of "strong emphasis" which we don't need here, this element is a label and doesn't need emphasis. `<b>` is only presentational, so at this point why not use just CSS?

Reply:
`<b>` actually does have semantic meaning as a "bring attention to" element.

>The `<b>` element [represents](https://www.w3.org/TR/html53/dom.html#represent) a span of text to which attention is being drawn for utilitarian purposes without conveying any extra importance and with no implication of an alternate voice or mood, such as key words in a document abstract, product names in a review, actionable words in interactive text-driven software, or an article lede.

https://www.w3.org/TR/html53/textlevel-semantics.html#the-b-element
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b

Reply:
> without conveying any extra importance .. etc.

> The b element should be used as a last resort when no other element is more appropriate. In particular, headings should use the h1 to h6 elements, stress emphasis should use the em element, importance should be denoted with the strong element, and text marked or highlighted should use the mark element.

To me, this means it's basically a presentational element to "highlight" some text. So why we should use an additional element? For presentation, CSS is more appropriate. Also, the current official recommendation is HTML 5.2. Instead. 5.3 is still a working draft.